.github/workflows/build.yml

```diff
@@ -0,0 +1,15 @@
+name: Deploy if not a version bump PR
+on:
+  push:
+    branches:
+      - master
+      - move-to-maven-and-github-actions
```

Comment:
Should this branch name be here?

Reply:
Nope, I took it out and forgot to commit it.